integration-cli/docker_cli_port_test.go

```diff
@@ -161,7 +175,7 @@ func assertPortList(c *testing.T, out string, expected []string) error {
 	// of the CLI used an incorrect output format for mappings on IPv6 addresses
 	// for example, "80/tcp -> :::80" instead of "80/tcp -> [::]:80".
 	oldFormat := func(mapping string) string {
-		old := strings.Replace(mapping, "-> [", "-> ", 1)
+		old := strings.Replace(mapping, "[", "", 1)
```

Comment:
I made the normalisation too strict in https://github.com/moby/moby/pull/42050, and forgot the "expected" format was sometimes passed as a "full" format (`80/tcp -> [::]:80`), and sometimes only the mapping (`[::]:80`); pushed this fix as a separate commit in this PR, but I can move to a separate PR if we think that makes more sense